ajax/libs/stampit/package.json

```diff
@@ -1,39 +1,37 @@
 {
   "name": "stampit",
-  "version": "2.1.0",
+  "version": "3.0.0",
   "description": "Create objects from reusable, composable behaviors.",
   "author": {
     "name": "Eric Elliott",
     "url": "https://ericelliottjs.com"
   },
+  "license": "MIT",
   "keywords": [
     "object",
     "prototype",
     "object oriented",
     "browser",
     "inheritance",
-    "oo",
+    "oop",
     "node",
     "factory",
-    "class"
+    "class",
+    "stamp"
   ],
   "repository": {
     "type": "git",
-    "url": "git://github.com/stampit-org/stampit-bower.git"
-  },
-  "dependencies": {
-    "lodash": "^3.9.1",
-    "supermixer": "^1.0.2"
-  },
-  "license": "MIT",
-  "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/stampit-org/stampit-bower.git",
-    "basePath": "",
-    "files": [
-      "stampit.js",
-      "stampit.min.js"
-    ]
+    "url": "git@github.com:stampit-org/stampit.git"
   },
-  "filename": "stampit.min.js"
+  "filename": "stampit.umd.min.js",
+  "npmName": "stampit",
+  "npmFileMap": [
+    {
+      "basePath": "/dist/",
+      "files": [
+        "stampit.umd.js",
+        "stampit.umd.min.js"
```

Comment:
@FredyC If the files under `dist` folder need to be added in cdnjs, let's use `"**/*"`. 
If you just want to add `stampit.umd.js` and `stampit.umd.min.js`, I suggest to use `"stampit.umd*"`. It will be more flexible. Thanks.